Solutions/Forescout/Parsers/ForescoutEvent.txt

```diff
@@ -0,0 +1,19 @@
+// This parser has been developed using Forescout Syslog Plugin version: v3.6
+// Usage instructions: 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name as RsaSecureIdEvent and specify Legacy Category.

```

Comment:
Function name should be as ForescoutEvent not  RsaSecureIdEvent.

Reply:
updated